### PR TITLE
Tests/ec34 login page

### PR DESCRIPTION
**Pull request by PaulineHavez**
### Tests for sign-In Page

**Files modified** : 

- [x] index.js of signIn Page
- [x] jest-config.ts in web-app
- [x] user.ts (in back-end)

**New Files** in folder tests : 

- [x] signin.dataset.tsx
- [x] signin.test.tsx
- [x] jest.setup.ts

**Do we need to update Database ?** : No

### CODE REVIEW

File: web-app/src/pages/login/sign-in/index.tsx [hunk 1]
```diff
@@ -114,6 +118,9 @@ export default function SignInPage() {
                 }}
               />
             </div>
+            <div className="w-full mt-4 text-md text-red-600">
+              {error && error.message}
```
Comment (JohannaStoicanescu) [inline web-app/src/pages/login/sign-in/index.tsx:122]: instead do `{error && <div className="w-full mt-4 text-md text-red-600">{error.mesage}</div>`

File: web-app/src/pages/login/sign-in/index.tsx
Comment (JonasJallet) [inline web-app/src/pages/login/sign-in/index.tsx:1]: like Johanna say just add condition for display div of error message

Comment (JonasJallet) [inline web-app/src/pages/login/sign-in/index.tsx:1]: like Johanna say just add condition for display div of error message

**Review by JonasJallet — CHANGES_REQUESTED** [review]

**Review by arnaudrenaud — APPROVED** [review]

**Review by JohannaStoicanescu — APPROVED** [review]

**Review by JonasJallet — APPROVED** [review]

**Pull request merged**